src/tracer/MARBL_tracers.F90

```diff
@@ -0,0 +1,2347 @@
+        else
+          fi = index(MARBL_instances%interior_tendency_forcings(m)%metadata%varname, &
+              'Restoring Inverse Timescale')
+          if (fi > 0) then
+            I_tau_count = I_tau_count + 1
+            CS%tracer_I_tau_ind(I_tau_count) = m
+          else
+            write(log_message, "(A,1X,A)") &
+                trim(MARBL_instances%interior_tendency_forcings(m)%metadata%varname), &
+                'is not a valid interior tendency forcing field name.'
+            call MOM_error(FATAL, log_message)
+          endif
+        endif
+    end select
+  enddo
+end subroutine configure_MARBL_tracers
+
+!> This subroutine is used to register tracer fields and subroutines
+!! to be used with MOM.
+function register_MARBL_tracers(HI, GV, US, param_file, CS, tr_Reg, restart_CS, MARBL_computes_chl)
+  type(hor_index_type),       intent(in) :: HI   !< A horizontal index type structure.
+  type(verticalGrid_type),    intent(in) :: GV   !< The ocean's vertical grid structure
+  type(unit_scale_type),      intent(in) :: US   !< A dimensional unit scaling type
+  type(param_file_type),      intent(in) :: param_file !< A structure to parse for run-time parameters
+  type(MARBL_tracers_CS),     pointer    :: CS   !< A pointer that is set to point to the control
+                                                 !! structure for this module
+  type(tracer_registry_type), pointer    :: tr_Reg !< A pointer that is set to point to the control
+                                                 !! structure for the tracer advection and diffusion module.
+  type(MOM_restart_CS), target, intent(inout) :: restart_CS !< MOM restart control struct
+  logical,                      intent(out)   :: MARBL_computes_chl  !< If MARBL is computing chlorophyll, MOM
+                                                                     !! may use it to compute SW penetration
+
+! Local variables
+! This include declares and sets the variable "version".
+# include "version_variable.h"
+  character(len=40)  :: mdl = "MARBL_tracers" ! This module's name.
+  character(len=256) :: log_message
+  character(len=200) :: inputdir ! The directory where the input files are.
+  character(len=48)  :: var_name ! The variable's name.
+  character(len=128) :: desc_name ! The variable's descriptor.
+  character(len=48)  :: units ! The variable's units.
+  character(len=96)  :: file_name ! file name for d14c (looped over three bands)
+  real, pointer :: tr_ptr(:,:,:) => NULL()
+  integer :: forcing_file_start_year
+  integer :: forcing_file_end_year
+  integer :: forcing_file_data_ref_year
+  integer :: forcing_file_model_ref_year
+  integer :: forcing_file_forcing_year
+  logical :: register_MARBL_tracers
+  logical :: restoring_has_edges, restoring_use_missing
+  logical :: restoring_timescale_has_edges, restoring_timescale_use_missing
+  real :: restoring_missing, restoring_timescale_missing
```

Comment:
Please add comments describing `restoring_missing`, and `restoring_timescale_missing` and their units as well as the other undocumented real variables throughout this module, as requested at https://github.com/mom-ocean/MOM6/wiki/Code-style-guide#documentation-in-code.

We have a way to actively come up with lists of all undocumented real variables anywhere in the directories under MOM6/src. If the comments are not added now by someone who knows what is going on, there is a risk that comments will be added later based on best guesses by a physical oceanographer who doesn't actually know what was intended in MARBL!

Reply:
`restoring_missing` and `restoring_timescale_missing` were addressed in 5402ce0, I still need to go back through the module and describe the other undocumented `real` declarations

Reply:
Rest of this comment was addressed in 9359ab7